modules/ROOT/pages/dw-core-functions-groupby.adoc

```diff
@@ -44,37 +52,77 @@ output application/json
       "name": "FooBar",
       "language": "Java"
     }
-  ]
-}
+]
 ----
 
-.Output
-[source,json,linenums]
+==== Output
+
+[source,JSON,linenums]
 ----
 {
-  "language": {
-    "Scala": [
-        {"name":"Bar", "language":"Scala"}
-      ],
-    "Java": [
-        {"name":"Foo", "language":"Java"},
-        {"name":"FooBar", "language":"Java"}
-      ]
-  }
+  "Scala": [
+    {
+      "name": "Bar",
+      "language": "Scala"
+    }
+  ],
+  "Java": [
+    {
+      "name": "Foo",
+      "language": "Java"
+    },
+    {
+      "name": "FooBar",
+      "language": "Java"
+    }
+  ]
 }
 ----
 
 
 [[groupby2]]
-== groupBy({ (K)?: V }, (value: V, key: K) -> R): { &#40;R&#41;: Array<T> }
+== groupBy({ (K)?: V }, (value: V, key: K) -> R): { (R): Array<T> }
+
+Groups elements of an object based on a supplied key.
+
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `object` | The object to group.
```

Comment:
We're not really grouping an object (I don't even know how you group one of something).